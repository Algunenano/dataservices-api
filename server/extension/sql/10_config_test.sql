-- Create a conf table
SELECT cdb_geocoder_server._config_set('cds', '{"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}');

-- Test key retrieval
SELECT cdb_geocoder_server._config_get('cds');

-- Test returns NULL if key doesn't exist
SELECT cdb_geocoder_server._config_get('no existe');
