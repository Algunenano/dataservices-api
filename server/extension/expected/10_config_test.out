-- Create a conf table
SELECT cdb_geocoder_server._config_set('cds', '{"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}');
 _config_set 
-------------
 
(1 row)

-- Test key retrieval
SELECT cdb_geocoder_server._config_get('cds');
                                _config_get                                 
----------------------------------------------------------------------------
 {"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}
(1 row)

-- Test returns NULL if key doesn't exist
SELECT cdb_geocoder_server._config_get('no existe');
 _config_get 
-------------
 
(1 row)

